library/std/src/lib.rs

```diff
@@ -89,7 +89,8 @@
 //! Check out the Rust contribution guidelines [here](
 //! https://rustc-dev-guide.rust-lang.org/contributing.html#writing-documentation).
 //! The source for this documentation can be found on
-//! [GitHub](https://github.com/rust-lang/rust).
+//! [GitHub](https://github.com/rust-lang/rust)
+//！(e.g. [here](https://github.com/rust-lang/rust/blob/master/library/std/src/lib.rs)).
```

Comment:
The character that you put here is a Unicode "Fullwidth Exclamation Mark", which is not an ASCII exclamation mark. So this line would not be treated as documentation by rustdoc. It is just a comment.

https://www.compart.com/en/unicode/U+FF01

![image](https://github.com/user-attachments/assets/37c64868-9d99-42d6-b16d-cd57b8a0ffed)

![image](https://github.com/user-attachments/assets/1bae725d-d8f3-41c4-b64c-0864d673b31c)